lesson03/util/util.go

```diff
@@ -0,0 +1,58 @@
+package util
+
+import (
+	"fmt"
+	"sort"
+)
+
+func FindAverage(arr []int) float64 {
+	if len(arr) == 0 {
+		fmt.Println("empty array")
+		return 1
+	}
+	sum := 0.0
+	for _, elem := range arr {
+		sum += float64(elem)
+	}
+	return sum / float64(len(arr))
+}
+
+func Max(sarr []string) string {
+	if len(sarr) == 0 {
+		return "empty array"
```

Comment:
Again, same as with the `FindAverage` function, this check might be unnecessary or at least should return an empty string in this case. Even more it might give false positive if your input array has a string "empty array" in it and it is not the longest string.